Parse/Platform/Objects/ParseObjectClassController.cs

```diff
@@ -112,6 +112,7 @@ public ParseObject Instantiate(string className, IServiceHub serviceHub)
             Classes.TryGetValue(className, out ParseObjectClass info);
             Mutex.ExitReadLock();
 
+            serviceHub = serviceHub ?? ParseClient.Instance;
```

Comment:
This looks suspicious. Why don't we just always use `ParseClient.Instance` if we know it cannot be null?